sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -549,6 +548,8 @@ Preconditions:
 
 Parameters:
 
+* `syclDevice` - Device this command will run on, if not a host task.
+
```

Comment:
I wonder if this is a bit confusing, but I'm not sure there's a better way. Perhaps an overload that doesn't take a device but that could easily lead to misuse.

There is `piEnqueueNativeKernel` which would make sense to have the device but it looks like only Intel OpenCL supports that at the moment.

Reply:
Overload without a device was also my initial thought. To make it a valid host task you would still have to call the specific function on a handler. Or what other misuse did you had in mind?